pom.xml

```diff
@@ -300,7 +300,26 @@ Brotli, Zstandard and ar, cpio, jar, tar, zip, dump, 7z, arj.
                 <exclude>org.apache.commons.compress.harmony.pack200.SegmentMethodVisitor</exclude>
                 <exclude>org.apache.commons.compress.harmony.pack200.SegmentAnnotationVisitor</exclude>
                 <exclude>org.apache.commons.compress.harmony.pack200.SegmentFieldVisitor</exclude>
+                <!-- Compress 1.28.0 -> 1.29.0: make TarUtils final -->
+                <exclude>org.apache.commons.compress.archivers.tar.TarUtils#parseFromPAX01SparseHeaders(java.lang.String)</exclude>
+                <exclude>org.apache.commons.compress.archivers.tar.TarUtils#parsePAX01SparseHeaders(java.lang.String)</exclude>
+                <exclude>org.apache.commons.compress.archivers.tar.TarUtils#parsePAX1XSparseHeaders(java.io.InputStream,int)</exclude>
+                <exclude>org.apache.commons.compress.archivers.tar.TarUtils#parsePaxHeaders(java.io.InputStream,java.util.List,java.util.Map)</exclude>
+                <exclude>org.apache.commons.compress.archivers.tar.TarUtils#parsePaxHeaders(java.io.InputStream,java.util.List,java.util.Map,long)</exclude>
               </excludes>
+              <!--
+                ~ JapiCmp does not support ignoring a specific compatibility change for a single class.
+                ~ As a workaround, we override the change globally.
+                ~
+                ~ TODO: Remove after release 1.29.0.
+                -->
+              <overrideCompatibilityChangeParameters>
```

Comment:
Obvious -1: We can't break binary compatibility for a trivial style/design change.

Reply:
OK, that makes sense; let's see what JApiCmp says about the ticket I created: https://github.com/siom79/japicmp/issues/426 

FWIW, the JLS' definition of binary compatible for final classes has changed from version to version, which I didn't bother to talk about in the ticket.